packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const COMPONENT_ID = 'loaders.progress';
+
+const { zdColorBlue600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
+
+const heights = {
+  small: 2,
+  large: 12,
+  default: 6
```

Comment:
We've been moving our APIs to use `medium` and then setting that size in `defaultProps`.

Reply:
That is also a much better pattern, I copied this from another component. I'll change that.